src/app/static/src/app/root.ts

```diff
@@ -78,6 +78,14 @@ export interface WarningsState {
     warnings: Warning[]
 }
 
+export enum STEPS  {
```

Comment:
We generally use pascal case rather than all caps for enums e.g. Language, LanguageMutation.

I guess we need this enum because it matches the 'locations' in the responses from hintr, which don't quite match the textKeys of the stepper steps?